synapse/lib/syntax.py

```diff
@@ -728,13 +749,36 @@ def query(self):
 
             self.ignore(whitespace)
 
-            #if self.nextstr('{'):
-                #self.offs += 1
-                #oper = self.query()
+            # if self.nextstr('{'):
```

Comment:
can we gut this unused subquery logic stub?

Reply:
yep